cupyx/jit/_builtin_funcs.py

```diff
@@ -94,14 +96,53 @@ def call(self, env, *args, **kwds):
 class SyncThreads(BuiltinFunc):
 
     def __call__(self):
-        """Calls ``__syncthreads()``
+        """Calls ``__syncthreads()``.
+
+        .. seealso:: `Synchronization functions`_
+
+        .. _Synchronization functions:
+            https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#synchronization-functions
         """
         super.__call__(self)
 
     def call_const(self, env):
         return Data('__syncthreads()', _cuda_types.void)
 
 
+class SyncWarp(BuiltinFunc):
+
+    def __call__(self, mask=0xffffffff):
```

Comment:
Could you fix `mask` to keyword only arugment?
```suggestion
    def __call__(self, *, mask=0xffffffff):
```

Reply:
Yes `mask` is meant to be a kw-only argument. Thanks @asi1024!